lib/features/popup-menu/PopupMenu.js

```diff
@@ -62,26 +66,32 @@ PopupMenu.$inject = [
  * Registers a popup menu provider
  *
  * @param  {String} id
+ * @param {Number} [priority=1000]
  * @param  {Object} provider
  *
  * @example
  *
  * popupMenu.registerProvider('myMenuID', {
```

Comment:
Maybe separate creating the provider from registering? That would make the example easier to comprehend in my opinion. Cf. https://github.com/bpmn-io/diagram-js/pull/431/commits/59681ee6ffc0398ff221130e3d34caa495017752